apps/fe/src/styles/globals.css

```
@@ -261,6 +284,11 @@
   height: 4.4rem;
 }
 
+@utility icon-60 {
```

Stylelint (17.11.1)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)